src/Propel/Generator/Util/BehaviorLocator.php

```diff
@@ -174,7 +174,7 @@ public function getBehavior($name)
      */
     private function getCoreBehavior($name)
     {
-        $phpName = NamingTool::toCamelCase($name);
+        $phpName = ucfirst(NamingTool::toCamelCase($name));
```

Comment:
shouldnt the naming tool already provide the correct name (otherwise it doesnt deserve its name)?